pkg/util/log/stdout_logger.go

```diff
@@ -253,10 +255,17 @@ func (s *stdoutLogger) Fatal(args ...interface{}) {
 	s.logMutex.Lock()
 	defer s.logMutex.Unlock()
 
-	s.writeMessage(fatalFn, fmt.Sprintln(args...))
+	msg := fmt.Sprintln(args...)
+
+	s.writeMessage(fatalFn, msg)
 	s.writeMessageToFileLogger(fatalFn, args...)
 
 	if s.fileLogger == nil {
+		analytics, err := analytics.GetAnalytics()
+		if err != nil {
```

Comment:
This check should be err == nil, otherwise send command event is never send or leads to a potential nil pointer